src/components/action-button.js

```diff
@@ -12,9 +12,10 @@ export const runActions = async actionList => {
 export default function ActionButton({ layout, button, Theme, app, context, senseNavigation, element }) {
   const { style, qStateName } = layout;
   const disabled = layout.useEnabledCondition && layout.enabledCondition === 0;
+  const inEditMode = context.permissions.indexOf('interact') === -1;
```

Comment:
@niekvanstaveren Does the interact permission always exist outside of edit mode or should this variable be called something else?

Reply:
That is a really good question. But changing the variable name into inInteractMode (or something like that) might be better then?

Reply:
I'll change it